src/UserRepository.js

```diff
@@ -6,6 +6,19 @@ class UserRepository {
     getUserData(id){
         return this.userData.find((user)=> user.id===id)
     }
+
+    getFriendData(friendIDs) {
+        let friendNames = []
+        const friends = friendIDs.filter(e => {
+                this.userData.forEach(t => {
+                if(t.id === e) {
+                    friendNames.push(t.name.split(' ')[0]) 
```

Comment:
This can be done with a filter and a map, and then our User class method to retrieve the first name.  I think we should refactor this before merging it into main.

Reply:
- fixed and refactored the get friends first name